File: LeetCode Solutions/Arrays 101/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0;
        for (int digit : nums){
            int numberOfDigits = 0;
            while (digit > 0){
                digit /= 10;
                numberOfDigits++;
            }
            if (numberOfDigits % 2 == 0){
                count++;
            }
        }
        return count;
    }
};

File: LeetCode Solutions/Learn/Arrays 101/Find Numbers with Even Number of Digits/Find Numbers with Even Number of Digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for digit in nums:
            numberOfDigits = 0
            while (digit > 0):
                digit = digit // 10
                numberOfDigits += 1
            if (numberOfDigits % 2 == 0):
                count += 1
        return count
        